_posts/2017-01-09-migrating-to-puppet4.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Migrating a Module to Puppet 4
+date: 2017-01-09
+github_username: bastelfreak
+twitter_username: BastelsBlog
+---
+
+
+We announced the deprecation of Puppet 3 in our modules
+[already](https://voxpupuli.org/blog/2016/12/22/putting-down-puppet-3/). There
+are a few steps needed to be taken before a Puppet-4-only release can be done:
+
+* Create a new branch which is called `puppet`
```

Comment:
I think we also want to state that a final puppet 3 release, announcing the deprecation of puppet 3 support in future versions, should be released prior to creating the branch.